API/chat/tests.py

```diff
@@ -670,13 +617,9 @@ def test_delete_message_that_does_not_exist(self):
             timestamp=timestamp
         )
 
-        qstring = urllib.urlencode({
-            'id': message.id + 1
-        })
-
         response = self.client.delete(
-            reverse('chat:message', args=(self.channel.name,)),
-            qstring
+            reverse('chat:message', args=(message.id + 1,)),
+            {}
```

Comment:
Having the same name would be fine if they have the same behaviour and just provide 2 sets of possible arguments.
The argument issue can be solved by using kwargs.
The endpoints don't have the same behaviour since delete/patch is only performed on one of the two so naming may as well reflect the difference.